misc bugs

jonathanhouge opened the pull request.

jonathanhouge committed:
handle when user is not logged in 'movies.js', removed console logs, added TODO

NerdRez committed:
Simple alert

jonathanhouge commented on landing_page/static/landing_page/js/movie.js:
```diff
@@ -5,7 +5,9 @@ var watchlist = false;
 var exclude = false;
 movieOverview(movie_data);
-fetchPreferences(movie_data.id); //TODO fetch this when getting the movie
+if(isLoggedIn){
```
julius please get prettier installed dude seriously

jonathanhouge reviewed (commented):
not a SINGLE pr can be merged into master until julius gets the prettier extension

jonathanhouge committed:
prettier in movie.js & css hover for onboarding import

jonathanhouge committed:
profile settings bar bug fix

jonathanhouge commented:
https://github.com/jonathanhouge/ReeldIn/pull/68 has been put in here, just to create a small bug center hub

farrahirodriguez commented:
> not a SINGLE pr can be merged into master until julius gets the prettier extension

![image](https://github.com/user-attachments/assets/34b97186-7d44-46bd-b549-247c2d49a11a)

jonathanhouge committed:
Merge remote-tracking branch 'origin' into movie.html-preferences-bug

NerdRez committed:
Added kebab case in profile files

NerdRez committed:
Minor code clarity

Didn't have to save the movie object so removed assignment and added comment explaining why this line is relevant

NerdRez reviewed (approved):
😎👍
